docs/modules/indexes/vectorstores/examples/mongodb_atlas_vector_search.ipynb

```diff
@@ -0,0 +1,168 @@
+{
```

Comment:
There is only one type of MongoDB server : it's the mongod process.
The vector search is not available in MongoDB. Only in MongoDB Atlas that abstracts Lucene behind the scene. Each vendor should introduce its own vectorstore since the underlying technologies can be very different and that it has been the current implementation. There is already a MongoDB Atlas vectorstore in the JS langchain library. IMO let's keep it that way.

Reply:
@P-E-B thanks for the feedback. I will add a separate vector store for the CosmosDB vector Store in both languages. I was just wondering if they could be combined here since there are so similar but that's ok.

Reply:
Perfect!